packages/recomponents/src/components/r-icon-button/r-icon-button.vue

```diff
@@ -57,9 +59,11 @@
                 // it requires
                 if (this.$slots['left-icon']) {
                     style['has-icon-left'] = true;
-                } else if (this.$slots['right-icon']) {
+                }
+                if (this.$slots['right-icon']) {
                     style['has-icon-right'] = true;
-                } else {
+                }
+                if (!this.$slots['right-icon'] && !this.$slots['right-icon']) {
```

Comment:
Is here a mistake?

Reply:
I did it that way because it wasn't capturing the styles properly If you added two icons only the `has-left-icon` class would be applied but not the `has-right-icon` and at the end, I added that check to count for a button just with the icon. If you check the `r-icon-button` on storybook you will notice there is no margin on the right icon.

Reply:
Thanks for the feedback :)